api/_utils.ts

```diff
@@ -302,10 +302,15 @@ export const queries: Record<number, () => QueryBase> = {
  * @returns An instance of the `RelayFeeCalculator` for the specific chain specified by `destinationChainId`
  */
 export const getRelayerFeeCalculator = (destinationChainId: number) => {
+  const queryFn = queries[destinationChainId];
+  if (typeof queryFn === undefined) {
+    throw new InputError("Invalid destination chain Id");
```

Comment:
For any troubleshooting purposes that might arise it could be nice to include the actual chain ID that was supplied here:

```suggestion
    throw new InputError(`Invalid destination chain Id: ${chainId}`);
```

Reply:
Agreed!